index.html

```diff
@@ -12,35 +12,172 @@
+            <div class="cardTop"></div>
+            <div class="cardBottom">
+              <h2 class="cardTitle">Multi-Post Stories Gain+Glory</h2>
+              <ul class="techs">
+                <li class="tech">Ruby on Rails</li>
+                <li class="tech">CSS</li>
+                <li class="tech">Javascript</li>
+                <li class="tech">html</li>
+              </ul>
+              <button class="btn">See Project</button>
+            </div>
+          </div>
+          <div class="projectCard">
+            <div class="cardTop"></div>
+            <div class="cardBottom">
+              <h2 class="cardTitle">Multi-Post Stories Gain+Glory</h2>
+              <ul class="techs">
+                <li class="tech">Ruby on Rails</li>
+                <li class="tech">CSS</li>
+                <li class="tech">Javascript</li>
+                <li class="tech">html</li>
+              </ul>
+              <button class="btn">See Project</button>
+            </div>
+          </div>
+          <div class="projectCard">
+            <div class="cardTop">
+            </div>
+            <div class="cardBottom">
+              <h2 class="cardTitle">Multi-Post Stories Gain+Glory</h2>
+              <ul class="techs">
+                <li class="tech">Ruby on Rails</li>
+                <li class="tech">CSS</li>
+                <li class="tech">Javascript</li>
+                <li class="tech">html</li>
+              </ul>
+              <button class="btn">See Project</button>
+            </div>
+          </div>
+          <div class="projectCard">
+            <div class="cardTop">
+            </div>
+            <div class="cardBottom">
+              <h2 class="cardTitle">Multi-Post Stories Gain+Glory</h2>
+              <ul class="techs">
+                <li class="tech">Ruby on Rails</li>
+                <li class="tech">CSS</li>
+                <li class="tech">Javascript</li>
+                <li class="tech">html</li>
+              </ul>
+              <button class="btn">See Project</button>
+            </div>
+          </div>
         </div>
       </section>
+      <div class="aboutbg">
+        <img src="./utils/icons/aboutme1desktop.svg" alt="" class="aboutbg1">
+        <img src="./utils/icons/aboutmemobile1.svg" alt="" class="aboutbg1mobile">
+        <section id="About" class="aboutme-section">
+          <div class="section-container">
+            <div class="aboutHeader">
+              <h1 class="sectionTitle">About Me</h1>
+              <p class="bioAboutme"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+              <button class="resumeBtn">Get My Reusme</button>
+              <img src="./utils/icons/aboutme2desktop.svg" alt="" class="aboutbg2">
+              <img src="./utils/icons/aboutme2mobile.svg" alt="" class="aboutbg2mobile">
```

Comment:
- [ ] Kindly don't use `img` tag for the background instead use the image as a background of the section in the CSS select by setting the` background` property